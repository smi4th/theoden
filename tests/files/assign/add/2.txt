a=1+2+3+4+5+6;
b=a+7+8+9;