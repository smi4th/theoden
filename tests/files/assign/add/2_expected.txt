---TEST OUTPUT---
---MEMORY---
a = ('int', 21)
b = ('int', 45)
---FUNCTIONS---
